src/components/SearchSubthoughts.js

```diff
@@ -77,7 +79,7 @@ const SearchSubthoughts = ({ search, searchLimit = DEFAULT_SEARCH_LIMIT, dispatc
     // must go into DOM to modify the parent li classname since we do not want the li to re-render
     ref={onRef}
   >
-    {!exists(search) ? <NewThought path={[]} label={`Create "${search}"`} value={search} type='button' /> : null}
+    {!exists(store.getState().thoughtIndex, search) ? <NewThought path={[]} label={`Create "${search}"`} value={search} type='button' /> : null}
```

Comment:
Since `thoughtIndex` is passed from `mapStateToProps`, you can use that instead of accessing state directly.